Description: libfdt-dev: Missing header file prevents the library usage
 device-tree-compiler (1.3.0-3) unstable; urgency=low
 .
   * libfdt-dev: Missing header file prevents the library usage
     Thanks Domenico Andreoli (Closes: #706137)
Author: Hector Oron <zumbi@debian.org>
Bug-Debian: http://bugs.debian.org/706137

--- device-tree-compiler-1.3.0.orig/libfdt/Makefile.libfdt
+++ device-tree-compiler-1.3.0/libfdt/Makefile.libfdt
@@ -4,7 +4,7 @@
 # be easily embeddable into other systems of Makefiles.
 #
 LIBFDT_soname = libfdt.$(SHAREDLIB_EXT).1
-LIBFDT_INCLUDES = fdt.h libfdt.h
+LIBFDT_INCLUDES = fdt.h libfdt.h libfdt_env.h
 LIBFDT_VERSION = version.lds
 LIBFDT_SRCS = fdt.c fdt_ro.c fdt_wip.c fdt_sw.c fdt_rw.c fdt_strerror.c
 LIBFDT_OBJS = $(LIBFDT_SRCS:%.c=%.o)
